docker/README.md

```diff
@@ -0,0 +1,18 @@
+Overview of how the ray images are built
+
+	 .─────────────────────────────────────────────────────.
```

Comment:
Can I suggest using a tree to visualize this stack?
```
nvidia/cuda
└── base-deps:gpu
    └── ray-deps:gpu
        └── ray:gpu
            └── ray-ml:gpu
ubuntu/focal
└── base-deps
    └── ray-deps
        └── ray
            └── ray-ml
```

Reply:
SGTM